SoftLayer/CLI/vlan/list.py

```diff
@@ -46,15 +52,21 @@ def cli(env, sortby, datacenter, number, name, limit):
                            vlan_number=number,
                            name=name,
                            limit=limit)
+
+    pods = mgr.get_pods_with_capabilities()
+
     for vlan in vlans:
         billing = 'Yes' if vlan.get('billingItem') else 'No'
 
         table.add_row([
             vlan.get('id'),
             vlan.get('vlanNumber'),
+            vlan.get('fullyQualifiedName'),
             vlan.get('name') or formatting.blank(),
-            get_gateway_firewall(vlan),
+            vlan.get('networkSpace').capitalize(),
             utils.lookup(vlan, 'primaryRouter', 'datacenter', 'name'),
+            get_pod_with_closed_announcement(vlan, pods),
```

Comment:
I think better add a some colour when the pod closed soon

Reply:
Cannot add color to a specific text for different reasons